src/commcare_cloud/fab/fabfile.py

```diff
@@ -398,6 +398,8 @@ def kill_stale_celery_workers():
 def deploy_formplayer():
     execute(announce_formplayer_deploy_start)
     execute(formplayer.build_formplayer, True)
+    if getattr(env, 'NEEDS_FORMPLAYER_RESTART', False):
```

Comment:
Any reason to not get rid of this environment variable, and restart formplayer where this is being set? https://github.com/dimagi/commcare-cloud/blob/bde9f4c7b3a1b517fd95f3b012091beb77701941/src/commcare_cloud/fab/operations/formplayer.py#L58-L62